samples/boards/96b_argonkey/sensors/prj.conf

```diff
@@ -5,5 +5,6 @@ CONFIG_I2C=y
 CONFIG_LED=y
 CONFIG_GPIO=y
 CONFIG_SENSOR=y
+CONFIG_HAS_STMEMSC=y
```

Comment:
Can it somehow be done directly in hal_st module ?
So that the dependency is resolved directly and simply by having the module in the manifest ?

Reply:
I moved back in the driver itself, where also the ```depends on ZEPHYR_ST_MODULE``` lies